packages/vitepress-plugin-changelog/src/index.ts

```diff
@@ -0,0 +1,106 @@
+import type { Plugin } from 'vite'
+import simpleGit from 'simple-git'
+import type { SimpleGit } from 'simple-git'
+import md5 from 'md5'
+import type { CommitInfo, ContributorInfo } from './types'
+
+let changeLogs: CommitInfo[] | undefined
+let git: SimpleGit
+
+const VirtualModuleID = 'virtual:nolebase-changelog'
+const ResolvedVirtualModuleId = `\0${VirtualModuleID}`
```

Comment:
According to the documentation of Vite.js[1], virtual module should use `virtual:` as prefix, and should resolve `\0` as prefix when resolving instead of the original implementation.

[1]: https://vitejs.dev/guide/api-plugin.html#virtual-modules-convention